data/governments.txt

```diff
@@ -1067,9 +1088,10 @@ government "Kor Mereti"
 	color .43 .48 .87
 	language "Korath"
 	
+	"default attitude" -0.01
 	"attitude toward"
-		"Kor Sestor" -.01
-		"Wanderer" -.01
+		"Author" 0
+		"Indigenous Lifeform (Subsidurial)" 0
```

Comment:
I look forward to when the (supposedly) friendly Mereti remain hostile to everyone but the Wanderers (and the player). It'll certainly make farming reputation with the Efreti easier, especially once their fleets expand towards the Wanderer settlements, if the Mereti still attack them.

Reply:
> would that carry over to existing saves?

It would not.